src/api.py

```diff
@@ -951,6 +954,80 @@ def HandleGetMessageDataByDestinationHash(self, params):
         data += ']}'
         return data
 
+    def HandleDisseminateRawObject(self, params):
+        if len(params) != 1:
+            raise APIError(0, 'I need 1 parameter!')
+        payload = self._decode(params[0], "hex")
+
+        # FIXME: obsolete function
```

Comment:
If I understand this right, we're moving from using shared to protocol. This first use of protocol starts that migration in api.py. So a comment here saying 'obsolete function' could be confusing.